src/Stan/Core/Id.hs

```diff
@@ -27,8 +28,9 @@ structures by using a phantom parameter.
 -}
 newtype Id a = Id
     { unId :: Text
-    } deriving stock (Show)
+    } deriving stock (Show, Generic)
       deriving newtype (Eq)
+      deriving anyclass (Hashable)
```

Comment:
I think we can derive `Hashable` using the `newtype` strategy here, since `Text` already has `Hashable` instance